### DISCUSSION TITLE
How can I modify the style of the box around a software system in the container view?

### DISCUSSION THREAD

**Discussion by mbraakhekke** [structurizr/lite#78] [Q&A] [answered]
### Question

In the default style, the box around a software system and the text in it in the container view is very hard to make out. See pic below. How can manipulate things like the font and line properties?

<img src= "https://github.com/structurizr/lite/assets/10126994/07469a88-cd9c-46f9-8253-e35c9ff01085" height="50%" width="50%">

**Comment by simonbrowndotje** [comment, marked as answer]
See [Structurizr - Help - Notation](https://structurizr.com/help/notation) for details.

**Reply by mbraakhekke** [reply]
Thanks! I missed that part of the docs. I had been searching through the [language reference](https://github.com/structurizr/dsl/blob/master/docs/language-reference.md), but couldn't figure it out from the info there.
